test/modules/packages/distributedIters/DistributedIters.chpl

```diff
@@ -296,6 +570,39 @@ where tag == iterKind.follower
   Helpers.
 */
 
+// Dynamic subrange calculation.
+/*
+  :arg c: The range from which to retrieve a dynamic subrange.
+  :type c: `range(?)`
+
+  :arg chunkSize: The chunk size. Must be positive.
+  :type chunkSize: `int`
+
+  :arg stage: The number of dynamic subranges to skip before returning a
+    dynamic subrange.
+  :type stage: `int`
+
+  :returns: A subrange of ``c``.
+
+  This function takes a range, a chunk size, and a stage, and simulates
+  performing OpenMP's dynamic schedule on the range with the given chunk
+  size until reaching the given stage. It then returns the subrange that the
+  dynamic schedule would have produced at that point.
+*/
+private proc dynamicSubrange(c:range(?),
+                             chunkSize:int,
+                             stage:int)
+{
+  const low:int = (c.low + (stage * chunkSize));
+  const potentialHigh:int = (low + (chunkSize - 1));
+  const cHigh = c.high;
+  const high:int = if potentialHigh <= cHigh
```

Comment:
Is this just ``min(potentialHigh, cHigh)`` ?

Reply:
... yes. Yes, it is!